changelog/unreleased/pr-17143.toml

```
@@ -0,0 +1,4 @@
+type = "a"
+message = "cluster name is configurable"
```

```suggestion
message = "data node cluster name is configurable"
```
should include data node since this is the general changelog